doc/src/conf.py

```diff
@@ -396,6 +396,7 @@
 # Enable links to other packages
 intersphinx_mapping = {
     'matplotlib': ('https://matplotlib.org/stable/', None),
+    'python': ('https://docs.python.org', None),
```

Comment:
Where is there a cross-reference to the Python docs?

Reply:
There used to be a link to a Python function. I removed the link, so I just removed the cross-reference.